lib/jelix/plugins/formwidget/time_html/time_html.formwidget.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @see        http://www.jelix.org
+ * @licence     http://www.gnu.org/licenses/lgpl.html GNU Lesser General Public Licence, see LICENCE file
+ */
+
+/**
+ * HTML form builder.
+ *
+ * @see http://developer.jelix.org/wiki/rfc/jforms-controls-plugins
+ */
+class time_htmlFormWidget extends \jelix\forms\HtmlWidget\WidgetBase
+{
+    protected function outputJs()
+    {
+        $ctrl = $this->ctrl;
+        $jFormsJsVarName = $this->builder->getjFormsJsVarName();
+        $js = 'c = new '.$jFormsJsVarName."ControlTime('".$ctrl->ref."', ".$this->escJsStr($ctrl->label).");\n";
+        $js .= "c.multiFields = true;\n";
+        $minTime = $ctrl->datatype->getFacet('minValue');
+        $maxTime = $ctrl->datatype->getFacet('maxValue');
+        if ($minTime) {
+            $js .= "c.min${minTime} = '".$minTime->toString(jDateTime::DB_TFORMAT)."';\n";
```

Comment:
why `c.min${minTime}` ? I think it should be `c.minTime`. Same issue for maxTime.